test/tests/performance/pbs_qsub_performance.py

```diff
@@ -53,30 +52,27 @@ def setUp(self):
 
     def test_submit_job(self):
         """
-         submission of 1000 jobs
-         before and after exporting variable
-         to check submission performance
+        submission of 1000 jobs
+        before and after exporting variable
+        to check submission performance
         """
 
-        start_time1 = time.time() * 1000
+        start_time1 = time.time()
         for i in range(1000):
-            j = Job(ROOT_USER)
+            j = Job(TEST_USER)
             self.server.submit(j)
-        end_time1 = time.time() * 1000
+        end_time1 = time.time()
 
-        for i in range(1000):
-            os.environ['VARIABLE' + str(i)] = 'b' * 2000
+        os.environ['VARIABLE'] = 'b' * 130000
 
-        start_time2 = time.time() * 1000
+        start_time2 = time.time()
         for i in range(1000):
-            j1 = Job(ROOT_USER)
+            j1 = Job(TEST_USER)
             self.server.submit(j1)
-        end_time2 = time.time() * 1000
+        end_time2 = time.time()
 
-        self.logger.info("sub. time before exporting env. variable in msec")
-        sub_time = int(end_time1 - start_time1)
+        sub_time1 = int(end_time1 - start_time1)
+        sub_time2 = int(end_time2 - start_time2)
 
-        self.logger.info(sub_time)
-        self.logger.info("sub. time after exporting env. variable in msec")
-        sub_time = int(end_time2 - start_time2)
-        self.logger.info(sub_time)
+        self.logger.info("sub. time without and with env is % s, % s sec"
+                         % (sub_time1, sub_time2))
```

Comment:
Could you remove the period after 'sub' and you have '% s'.  I think you want '%d'.  Also, since you have chosen to leave this as a performance test, please move the test plan to the performance directory.